src/index.js

```diff
@@ -63,6 +63,7 @@ class Client {
 
     this.agentOptions = agentOptions;
     this.auth = (password || username) && { pass: password, user: username };
+    this.hasNamedParametersSupport = true;
```

Comment:
@pedrobranco I don't think the default should be `true`. Without passing a specific version, we can't be sure of that, so it seems rather imprudent to simply enable this for all clients.